Internal/Deploy.ps1

```diff
@@ -15,323 +16,197 @@ $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-S
-                $name = $_.FullName
-                Write-Host "Remove $name"
-                if ($_.PSIsContainer) {
-                    Remove-Item $name -Force -Recurse
-                }
-                else {
-                    Remove-Item $name -Force
-                }
-            }
-            
-            Write-Host -ForegroundColor Yellow "Collecting from $repo"
-            Get-ChildItem -Path $dstPath -Recurse -File -Force | Where-Object { $_.name -notlike '*.copy.md' } | ForEach-Object {
-                $dstFile = $_.FullName
-                $srcFile = $srcPath + $dstFile.Substring($dstPath.Length)
-                $srcFilePath = [System.IO.Path]::GetDirectoryName($srcFile)
-                if (!(Test-Path $srcFilePath)) {
-                    New-Item $srcFilePath -ItemType Directory | Out-Null
-                }
-                Write-Host "$dstFile -> $srcFile"
-                $lines = ([string](Get-ContentLF -path $dstFile)).Split("`n")
-                "actionsRepo","perTenantExtensionRepo","appSourceAppRepo" | ForEach-Object {
-                    $regex = "^(.*)$($config.githubOwner)\/$($config."$_")(.*)$($config.branch)(.*)$"
-                    $replace = "`$1$($originalOwnerAndRepo."$_")`$2$originalBranch`$3"
-                    $lines = $lines | ForEach-Object { $_ -replace $regex, $replace }
-                }
-                if ($_.Name -eq "AL-Go-Helper.ps1") {
-                    $lines = $lines | ForEach-Object { $_ -replace '^(\s*)\$defaultBcContainerHelperVersion(\s*)=(\s*)"(.*)"(.*)$', "`$1`$defaultBcContainerHelperVersion`$2=`$3""""`$5" }
-                }
-                [System.IO.File]::WriteAllText($srcFile, "$($lines -join "`n")`n")
-            }
-        }
-        Set-Location $baseRepoPath
-
-        if ($github) {
-            $serverUrl = "https://$($user.login):$token@github.com/$($srcOwnerAndRepo).git"
-        }
-        else {
-            $serverUrl = "https://github.com/$($srcOwnerAndRepo).git"
-        }
+    $dstOwnerAndRepo = @{
+        "perTenantExtensionRepo" = "$($config.githubOwner)/$($config.perTenantExtensionRepo)"
+        "appSourceAppRepo" = "$($config.githubOwner)/$($config.appSourceAppRepo)"
+    }
 
-        $commitMessage = "Collect changes from $($config.githubOwner)/*@$($config.branch)"
-        invoke-git add *
-        invoke-git commit --allow-empty -m "'$commitMessage'"
-        if ($baseRepoBranch) {
-            invoke-git push -u $serverUrl $baseRepoBranch
-            invoke-gh pr create --fill --head $baseRepoBranch --repo $srcOwnerAndRepo --base $ENV:GITHUB_REF_NAME
-            invoke-git checkout $algoBranch
-        }
-        else {
-            invoke-git push $serverUrl
+    if ($config.branch -eq 'preview') {
+        # When deploying to preview, we are NOT going to deploy to a branch in the AL-Go-Actions repository
+        # Instead, we are going to have AL-Go-PTE and AL-Go-Actions point directly to the SHA in AL-Go

```

Comment:
Wording is bit off, isn't it?

Something like...
```suggestion
        # Instead, we are going to have AL-Go-PTE and AL-Go-AppSource point directly to the SHA in AL-Go-Actions

```

Reply:
You are right about AppSource, but it will actually point to the SHA in AL-Go - not AL-Go-Actions.